### PR TITLE
Update Makefile to export EMULATOR

### PR DESCRIPTION
**Pull request by jordanhubbard**
Default value of EMULATOR is not passed in the environment in newer versions of gmake without an explicit export.

### CODE REVIEW

**Comment by ams** [comment]
What is "newer GNU make" here?  The last release is 4.4 (ignoring the bug fix 4.4.1) which is from 2022 ... which doesn't seem "recent".

**Comment by jordanhubbard** [comment]
Probably a poor choice of words on my part - I thought it was a new behavior when, in fact, it is an *old* behavior - macOS stopped updating all GPLv3 tools some time ago:

***@***.***> make -v
GNU Make 3.81

What occurs without the work-around I committed:

***@***.***> make
git submodule sync --recursive `dirname tools/pdp6/.gitignore`
Synchronizing submodule url for 'tools/pdp6'
git submodule update --recursive --init `dirname tools/pdp6/.gitignore`
build/stamp.sh build/timestamps.txt
mkdir -p out/pdp10-ka/stamp
touch out/pdp10-ka/stamp/touch
mkdir -p out/pdp10-ka/system
cp build/pdp10-ka/config.* out/pdp10-ka/system
mkdir -p out/pdp10-ka
tools/itstar/itstar -cf out/pdp10-ka/ka-minsys.tape -C bin/ka10 _ sys
tools/itstar/itstar -rf out/pdp10-ka/ka-minsys.tape -C bin/minsys sys
mkdir -p out/pdp10-ka
tools/itstar/itstar -cf out/pdp10-ka/minsrc.tape -C src midas system sysen1/ddt.1548 syseng/datime.75 syseng/lsrtns.69 syseng/msgs.47 syseng/ntsddt.n79h kshack/nsalv.261 syseng/format.305 syseng/rfn.13 kshack/ksfedr.146 syseng/dump.448 sysnet/netwrk.266
tools/itstar/itstar -rf out/pdp10-ka/minsrc.tape -C out/pdp10-ka system
cd bin/ka10/boot; ../../../tools/dasm/magfrm magdmp.bin @.ddt salv.bin > ../../../out/pdp10-ka/magdmp.tap
File @.ddt -> @ ddt
File salv.bin -> salv bin
ln -s build/pdp10-ka/start 
mkdir -p out/pdp10-ka/stamp
cp build/mchn/KA/run out/pdp10-ka/run
mkdir -p out/pdp10-ka/stamp
touch out/pdp10-ka/stamp/pdp10
mkdir -p out/pdp10-ka/syshst
sed -e 's/%IP%/192.168.1.100/' \
	    -e 's/%HOSTNAME%/DB-ITS.EXAMPLE.COM/' < build/h3text.2018 > out/pdp10-ka/syshst/h3text.2018
cat conf/hosts >> out/pdp10-ka/syshst/h3text.2018
mkdir -p out/pdp10-ka
rm -f -f src/*/*~
tools/itstar/itstar -cf out/pdp10-ka/sources.tape -C src syseng sysen1 sysen2 sysen3 sysnet kshack dragon channa _teco_ emacs emacs1 rms klh syshst sra mrc ksc eak gren bawden _mail_ l lisp libdoc comlap lspsrc nilcom rwk chprog rg inquir acount gz sys decsys ecc alan sail kcc kcc_sy c games archy dcp spcwar rwg libmax rat z emaxim rz maxtul aljabr cffk das ell ellen jim jm jpg macrak maxdoc maxsrc mrg munfas paulw reh rlb rlb% share tensor transl wgd zz graphs lmlib pratt quux scheme gsb ejs mudsys draw wl taa tj6 budd sharem ucode rvb kldcp math as imsrc gls demo macsym lmcons dmcg hibou agb gt40 rug maeda ms kle aap common fonts lcf 11logo kmp info aplogo bkph bbn pdp11 chsncp sca music1 moon teach ken lmio1 llogo a2deh chsgtv clib sys3 lmio turnip mits_s rab stan_k bs cstacy kp dcp2 -pics- victor imlac rjl mb bh lars drnil radia gjd maint bolio cent shrdlu vis cbf digest prs jsf decus bsg muds54 hello rrs 2500 minsky danny survey librm3 librm4 klotz atlogo clusys cprog r eb cpm mini nova sits nlogo bee gld mprog2 cfs libmud librm1 librm2 mprog mprog1 mudbug mudsav _batch combat mits_b minits spacy _xgpr_
tools/itstar/itstar -rf out/pdp10-ka/sources.tape -C doc info _info_ sysdoc sysnet syshst kshack _teco_ emacs emacs1 c kcc chprog sail draw wl pc tj6 share _glpr_ _xgpr_ inquir mudman system xfont maxout ucode moon acount alan channa fonts games graphs humor kldcp libdoc lisp _mail_ midas quux scheme manual wp chess ms macdoc aplogo _temp_ pdp11 chsncp cbf rug bawden llogo eak clib teach pcnet combat pdl minits mits_s chaos hal -pics- imlac maint cent ksc klh digest prs decus bsg madman hur lmdoc rrs danny netwrk klotz hello clu r mini nova sits jay rjl nlogo mprog2 mudbug cfs hudini
tools/itstar/itstar -rf out/pdp10-ka/sources.tape -C bin sys sys1 sys2 emacs _teco_ lisp liblsp alan inquir sail comlap c decsys graphs draw datdrw fonts fonts1 fonts2 games macsym maint _www_ gt40 llogo bawden sysbin -pics- lmman shrdlu imlac pdp10 madman survey rrs clu clucmp rws mini mudsav mudsys libmud librm1 librm2 librm3 librm4 mbprog mprog1 mprog mprog2 mudbug mudtmp _batch
tools/itstar/itstar -rf out/pdp10-ka/sources.tape -C out/pdp10-ka syshst
expect -f build/pdp10-ka/build.tcl 192.168.1.100 192.168.0.45
no such variable
    (read trace on "env(EMULATOR)")
    invoked from within
"set out "out/$env(EMULATOR)""
    (file "build/pdp10-ka/build.tcl" line 2)
make: *** [out/pdp10-ka/rp03.2] Error 1

Using "env EMULATOR=pdp10-ka make” works around this problem but, of course, that also requires the user to know that this environment variable is essential and wouldn’t understand the error if they weren’t familiar with Tcl.

- Jordan

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Some important environment variables
+t # Some important environment variables
```
Comment (larsbrinkhoff) [inline Makefile:1]: Stray "t" here.  Please *amend* the commit, and force-push to the branch.

**Comment by larsbrinkhoff** [comment]
All builds now fail with this message: `Makefile:1: *** missing separator.  Stop.`

**Comment by jordanhubbard** [comment]
Yes, a syntax error crept in when a puppy bounced on the keyboard during the PR (seriously!) - fixing this now and also simplifying the previous commit.

**Comment by larsbrinkhoff** [comment]
Thank you.  Please squash this into a single commit.

**Review by larsbrinkhoff — CHANGES_REQUESTED** [review]
Please squash into a single commit and rebase.

**Pull request closed without merging**